template/frontend/.env.example

```diff
@@ -0,0 +1,3 @@
+TEST_URL=http://localhost:3000/
+TEST_EMAIL=test@email.com
+TEST_PASSWORD=password
```

Comment:
awesome :-) 

so as a user, should we copy this `.env.example` to `.env`? and then run `yarn test`?

Let's add something in `template/README.md` for this. Maybe the following (in the "Available Scripts" section):

```
### `cd frontend && yarn test`

Runs end-to-end tests.
```